web/book/tests/documentation/snapshots/documentation__book__queries__functions__piping__2.snap

```diff
@@ -4,9 +4,7 @@ expression: "let fahrenheit_to_celsius = temp -> (temp - 32) / 1.8\nlet interp =
 ---
 SELECT
   *,
-  (
-    (((temp_c - 32) * 1.0 / 1.8) - 0) * 1.0 / (100 - 0)
-  ) AS boiling_proportion
+  ((((temp_c - 32) / 1.8) - 0) / (100 - 0)) AS boiling_proportion
```

Comment:
I know we removed arithmetic from PRQL, but this would be cool to simplify

("cool", not necessarily "important"!)